corgidrp/ops.py

```diff
@@ -0,0 +1,16 @@
+
+import corgidrp.caldb as caldb
+import corgidrp.walker as walker
+import argparse
+
+
+def step_1_initialize():
```

Comment:
Let's add some docstrings to these three functions, especially since they are outwards facing.